firebase/index.ts

```diff
@@ -0,0 +1,22 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { getFirestore } from "firebase/firestore";
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+const firebaseConfig = {
+apiKey: "AIzaSyCf9QLPlik5QJcAtDgY_5P-VbwD2WeKjZ0",
+authDomain: "[next-firebase-todo-b0b9e.firebaseapp.com](http://next-firebase-todo-b0b9e.firebaseapp.com/)",
+projectId: "next-firebase-todo-b0b9e",
+storageBucket: "[next-firebase-todo-b0b9e.appspot.com](http://next-firebase-todo-b0b9e.appspot.com/)",
+messagingSenderId: "150417331472",
+appId: "1:150417331472:web:818ffb23f32e3a1bcfc195"
+};
```

Comment:
Please add these to a .env file and then call these values through a constant file to this file. Basically, have a folder called "constants" and in that have a file called "firebase.ts" and in this file have something like:

export const apiKey = [call the .env variable]

A couple of useful links for you to get an idea if required:

1. https://blog.logrocket.com/customizing-environment-variables-next-js-13/
2. https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables

Reply:
`const firebaseConfig = {
  apiKey: process.env.NEXT_PUBLIC_FIREBASE_API_KEY,
  authDomain: process.env.NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN,
  projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
  storageBucket: process.env.NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET,
  messagingSenderId: process.env.NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID,
  appId: process.env.NEXT_PUBLIC_FIREBASE_APP_ID,
};` imported sensitive values as env variables

Reply:
committed the changes as well

Reply:
Perfect.